simple-api/README.md

```diff
@@ -1,2 +1,165 @@
+Enclave Name:                         kurtosis-enclave
+Enclave Status:                       RUNNING
+Creation Time:                        Mon, 27 Feb 2023 16:21:36 EST
+API Container Status:                 RUNNING
+API Container Host GRPC Port:         127.0.0.1:50122
+API Container Host GRPC Proxy Port:   127.0.0.1:50123
+
+========================================== User Services ==========================================
+UUID           Name                  Ports                                      Status
+ddf04ab5ba64   kurtosis-simple-api   http: 8080/tcp -> http://127.0.0.1:50137   RUNNING
+```
+
+We can see that the `kurtosis-simple-api` service is running on port 8080 inside the enclave and that it's mapped to port `50137` outside of Docker.
+This means that we can interact with the service using port `50137` as in this example (assuming `jq` is installed):
+
+```shell
+$ curl 'http://localhost:50137/api/v1/logs?service_uuid=13972c20f7d34b39b2ee75036778c9e3' | jq 
+[
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "Initializing DynamoDB Local with the following configuration:"
+  },
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "Port:\t8000"
+  },
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "InMemory:\ttrue"
+  },
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "DbPath:\tnull"
+  },
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "SharedDb:\tfalse"
+  },
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "shouldDelayTransientStatuses:\tfalse"
+  },
+  {
+    "service_name": "dynamodb",
+    "service_uuid": "13972c20f7d34b39b2ee75036778c9e3",
+    "log": "CorsParams:\tnull"
+  }
+]
+```
+
+Note that services _inside_ the enclave must still use port 8080 to interact with the Kurtosis Simple API.
+
+6. The enclave can be removed by running this command:
+
+```shell
+$ kurtosis enclave rm kurtosis-enclave --force
+```
+
+or by using the more indiscriminate alternative that will remove _all_ enclaves:
+
+```shell
+$ kurtosis clean -a
+```
+## Running Kurtosis by building Kurtosis Simple API locally
+
+If you are unable to use Dockerhub, you can build the Kurtosis Simple API locally by running this command:
+
+```shell
+$ bash scripts/build.sh 
+```
+
+This will build and push the Kurtosis Simple API to your local Docker Engine. 
+Note, if you build your image locally and then run the Kurtosis Simple API Starlark script (as described in the section above),
+the Starlark engine will download the latest `kurtosistech/kurtosis-simple-api` image from Dockerhub and 
+move the name and tag from your local build to this build from Dockerhub, thus leaving your local build unnamed and dangling.
```

Comment:
Wait, is that true? Just to make sure I understand. Let's say I am in this repo and I run:
```
./script/build.sh
```
After that, I'll have a new image in my local docker named `kurtosistech/kurtosis-simple-api:latest`
If after that I run my starlark package, since the image in my docker already exist, the CLI will not bother downloading it from docker hub, it will just use the existing one.

I suppose you've tested that, so I'm wondering if there's a bug somewhere here. If I understand correctly, the behaviour you're describing here is really not what we want, so maybe there's a bug that needs to be fixed

Reply:
haha @gbouv I did test it and this was the behavior I observed. I assumed that this was the way it was supposed to work, but also struggled with coming to terms with that since it seems counterproductive. Maybe we should look at this together to make sure I'm indeed doing this right?

Reply:
Ohh, then I think you're onto something @adschwartz, this definitely sounds like a bug. I'm happy to quickly look into it if you don't want to allocate some of your time to this specific issue. BUt also happy to look into it together if you're curious. If the former feel free to create a ticket an assign it to me, otherwise just ping me and we can quickly chat about it

Reply:
But I think anyway you should rephrase that to be in sync with the desired behaviour (i.e. we pull local images first, and fallback to docker hub download if not found), such that you don't have to update the wording here once the bug is fixed